install/linux/build-rpm.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 import argparse
+import getpass
```

Comment:
Why is this import required? 🤔

Reply:
Whoops, that's no longer required, Removed it in bdf0204dd33730fc294c82bcd934a35f940da6e2.